### ISSUE TITLE
prettify version output

### ISSUE THREAD

**Issue by wyardley** [hashicorp/terraform#28417]
While I get the specific case mentioned in #26975, would hashicorp accept a PR to update the format to something like
```
Terraform v0.15.0 (darwin_amd64)
```
vs
```
Terraform v0.15.0
on darwin_amd64
```

I'm happy to make such a PR, but obviously, won't do it if there's no interest.

I can see the argument about using a `head -1`, but the current format is not really very aesthetically pleasing, and could cause its own set of problems. It should be relatively trivial to use awk, or else use `tf version -json` if grabbing the version string only programmatically is needed.

**Comment by apparentlymart** [comment]
Hi @wyardley,

What you are suggesting here is more or less what I had as my first pass of that other PR, but as you saw I ended up changing it as a compromise. I think that compromise still applies, given that we're talking about a subjective presentation thing vs. potentially upsetting existing integrations.

If there were a reason to do this other than an individual's sense of aesthetics (which I share with you, but still...) then I expect we'd consider it, but it's hard to justify creating annoyance for existing users _just_ to make the output a little more compact. This output tends to be longer than this in most cases anyway, because it'll also list out the providers that are selected in the dependency lock file, and so (again, totally subjectively) I considered it _good enough_ to meet the use case I described, which was literally just to get people to tell us which platform they are using when they open a bug report, without us having to prompt them first.
